src/resources/auth.ts

```diff
@@ -162,6 +163,26 @@ export class Auth extends BaseResource {
     }
   }
 
+  /**
+   * Create a new authorization request and get a new unique login url. 
+   * Used only for hosted authentication. 
+   * This is the initial step requested from the server side to issue a new login url. 
+   * @param HostedAuthRequest params to initiate hosted auth request
+   */
+  public async hostedAuth(payload: HostedAuthRequest): Promise<boolean> {
+    await this.apiClient.request<EmptyResponse>(
+      {
+        method: 'POST',
+        path: `/v3/connect/revoke`,
+        body: payload
+      },
+      {
+        responseSchema: EmptyResponseSchema,
```

Comment:
should this be HostedAuthResponseSchema or HostedAuthSchema? I don't see where those two schemas are used at all

Reply:
You are right, made the changes